doc/netplan-yaml.md

```diff
@@ -1268,12 +1286,30 @@ The specific settings for bridges are defined below.
     > renderer is used. If no time suffix is specified, the value will be
     > interpreted as seconds.
 
-  - **path-cost** (scalar)
+  - **path-cost** (mapping)
 
     > Set the cost of a path on the bridge. Faster interfaces should have
     > a lower cost. This allows a finer control on the network topology
     > so that the fastest paths are available whenever possible.
```

Comment:
```suggestion
    > Set the per-interface cost of a path on the bridge. Faster interfaces
    > should have a lower cost. This allows a finer control on the network
    > topology so that the fastest paths are available whenever possible.
```